src/urllib3/util/ssl_.py

```diff
@@ -67,6 +67,20 @@ def _is_ge_openssl_v1_1_1(
     )
     PROTOCOL_SSLv23 = PROTOCOL_TLS
 
+    # https://github.com/urllib3/urllib3/issues/2192#issuecomment-821832963
```

Comment:
Can we put a brief description of why we're doing this so readers don't have to click into a URL for context?

Something short like "SSLContext.hostname_checks_common_name = False didn't work before CPython 3.8.9, 3.9.3, and 3.10"

Reply:
Thanks, added.